apps/ui/src/components/views/board-view/dialogs/backlog-plan-dialog.tsx

```
@@ -0,0 +1,418 @@
+              <div className="flex items-center gap-2 text-sm text-muted-foreground bg-muted/50 rounded-lg p-3">
+                <Loader2 className="w-4 h-4 animate-spin" />A plan is currently being generated in
+                the background...
+              </div>
+            )}
+          </div>
+        );
+
+      case 'review':
+        if (!pendingPlanResult) return null;
+
+        const additions = pendingPlanResult.changes.filter((c) => c.type === 'add');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)